docs/en/observability/application-logs.asciidoc

```diff
@@ -0,0 +1,219 @@
+parsing your logs into a structured object at ingest time.
+
+Why is structure so important in logs? So you can search and aggregate on individual fields within your logs.
+
+
+[float]
+[[ingest-application-logs]]
+== Ingest application logs
+
+There are four ways to ship your application logs to the {stack}.
+Your use case will determine the method that's right for you.
+
+[float]
+=== Plaintext logs with Filebeat
+
+**Requirements**
+
+* The Elastic APM agent for your programming language.
+* Raw, plain-text application logs
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, filter and correlate multi-line logs, and forwards them to {es}
+
+**Pros**
+
+* All programming languages/frameworks are supported
+
+**Cons**
+
+* Must parse application logs to be useful—meaning writing and maintaining Grok patterns and spending CPU cycles on parsing
+
+Learn more: <<plaintext-with-filebeat>>
+
+[float]
+=== ECS loggers with Filebeat
+
+Elastic Common Schema (ECS) loggers format your logs into ECS-compatible JSON,
+which removes the need to manually parse logs.
+
+**Requirements**
+
+* The Elastic APM agent for your programming language
+* The Elastic ECS logger for your language or framework
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, and forwards them to {es}
+
+**Pros**
+
+* Popular logging frameworks supported
+* Simplicity: no manual parsing
+* Decently human-readable JSON structure
+* APM Log correlation
+* Resilient in case of outages
+
+**Cons**
+
+* Not all frameworks are supported
+
+Learn more: <<ecs-with-filebeat>>
+
+[float]
+=== APM agent ECS reformatting with Filebeat
+
+Elastic APM agents can automatically reformat application logs to Elastic Common Schema (ECS) format
+without needing to add an ECS logger dependency.
+
+**Requirements**
+
+* The Elastic APM agent for your programming language
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, and forwards them to {es}
+
+**Pros**
+
+**Cons**
+
+**Supported APM agents/languages**
+
+* Ruby
+* Java
+
+Learn more: <<ecs-reformatting-with-filebeat>>
+
+[float]
+=== APM agent direct logs
+
+Elastic APM agents can automatically capture and send logs directly to APM Server--enabling you to
+easily ingest log events without needing a separate log shipper like {filebeat}.
+
+**Requirements**
+
+The Elastic APM agent for your programming language.
+
+**Pros**
+
+**Cons**
+
+**Supported APM agents/languages**
+
+* Java
+
+Learn more: <<apm-agent-direct>>
+
+[float]
+[[log-correlation]]
+== Log correlation
+
+Elastic APM integrates with the most popular logging frameworks in each programming language.
+These integrations inject trace, transaction, and error ID fields into your application's log records,
+allowing you correlate and easily switch between log and trace events.
+
+[float]
+=== Unstructured logs vs. ECS logs
```

Comment:
I think it might be better to frame this as "plaintext" vs "ECS logs" so we can simplify and skip the "structured but not ECS" format.

Reply:
👍 Good idea.

Reply:
Fixed in https://github.com/elastic/observability-docs/pull/2452/commits/ef578dbcbb599be0dc52accf37161e4f587f113f.